code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -378,6 +381,60 @@
 
 	return dat
 
+/mob/living/silicon/robot/proc/toggle_overclock()
+	set category = "Robot Commands"
+	set name = "Toggle Overclock"
+	set desc = "Enable an overclocking of your systems, greatly increasing the power available to your modules."
+
+	if(overclockavailable == 1)
+		if(overclocked == 0)
+			overclocked = 1
+			ToggleOverClock(src)
+			usr << "You enable the overclock mode enhancing and unlocking several modules but increasing power usage greatly."
+		else
+			overclocked = 0
+			ToggleOverClock(src)
+			usr << "You disable the overclock mode."
+
+/mob/living/silicon/robot/proc/ToggleOverClock()

```

Comment:
This should be:
`/mob/living/silicon/robot/proc/ToggleOverClock(var/mob/living/silicon/robot/R)`

`usr` should be used in a proc other than procs as verbs and verbs. This is neither. don't forget to check for R at the very beginning of this proc.